cmake/Phylanx_Documentation.cmake

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2018 Bibek Wagle
+#
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+option(PHYLANX_BUILD_DOCS "Build Phylanx Documentation" OFF)
```

Comment:
You might want to consider using `phylanx_option()` instead (see here for some examples: https://github.com/STEllAR-GROUP/phylanx/blob/master/CMakeLists.txt#L86-L104).